storage_devices/disk.py

```diff
@@ -157,50 +165,41 @@ def unplug(self):
 
     @staticmethod
     def plug_all_disks():
-        TestRun.executor.run_expect_success(NvmeDisk.plug_all_command)
-        TestRun.executor.run_expect_success(SataDisk.plug_all_command)
+        for disk_type in DiskRegistry.driver_types:
+            TestRun.executor.run_expect_success(disk_type.plug_all_command)
```

Comment:
`plug_all_command` shouldn't be accessed outside of the class. Instead each of of the subclasses should implement a method `plug_all()`

Reply:
Also, instead of looping over the disk types here, I suggest `DiskTypeRegistry` implements a method `plug_all_disks()` or sth like this